ta/os_test/pauth_a64.S

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Linaro Limited
+ */
+
+#include <asm.S>
+
+FUNC corrupt_pac , :
```

Comment:
It could be nice to have the prototype of the C function in a comment above as a reminder of what the caller can expect. Perhaps even a short description of what the function does as I guess most people are not very familiar with the instructions `paciasp` and `autiasp`.

Reply:
Sure, will add